routes/auth/logout.js

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+import controllers from 'controllers';
+import isAuth from 'middlewares/isAuth';
```

Comment:
Also remove here and if `refreshToken` file

Reply:
Done.